src/js/store/middleware/feature/form.js

```diff
@@ -226,6 +265,46 @@ const handleClear = (store, next) => {
   ])
 }
 
+const handlePersistInLocalStorage = (store) => {
+  const state = store.getState()
+  const key = localStorageKey(state)
+  const storageState = {
+    schema: state.form.schema,
+    fields: state.form.fields,
+  }
+  window.localStorage.setItem(key, JSON.stringify(storageState))
+}
+
+const handleClearLocalStorage = (store) => {
+  const state = store.getState()
+  const key = localStorageKey(state)
+  window.localStorage.removeItem(key)
```

Comment:
In response to `CLEAR_LOCAL_STORAGE` actions we remove this form's data from local storage. This action is triggered either when we _reject_ local storage (automatically or because the user said "no" to the prompt) or when the user successfully submits their work.